fix: env vars for middleware (#205)
plentymarkets/plentyshop-pwa: docs/changelog/changelog_en.md

## v1.1.0 (XX.XX.2023) <a href="https://github.com/plentymarkets/plentyshop-pwa/compare/v1.0.0...v1.0.1" target="_blank" rel="noopener"><b>Overview of all changes</b></a>

### TODO

> - [Middleware](./docs/config/middleware.md) API_ENDPOINT must now be maintained via an .env file under apps/server.

Reviewer: ```suggestion
- [Middleware](./docs/config/middleware.md) `API_ENDPOINT` now has to be maintained via an `.env` file under `apps/server`.
```
